src/soilbiogeochem/SoilBiogeochemDecompCascadeMIMICSMod.F90

```diff
@@ -1,15 +1,15 @@
-module SoilBiogeochemDecompCascadeBGCMod
+module SoilBiogeochemDecompCascadeMIMICSMod
 
   !-----------------------------------------------------------------------
   ! !DESCRIPTION:
   ! Sets the coeffiecients used in the decomposition cascade submodel.  
-  ! This uses the CENTURY/BGC parameters
+  ! This uses the MIMICS parameters.
   !
   ! !USES:
   use shr_kind_mod                       , only : r8 => shr_kind_r8
   use shr_const_mod                      , only : SHR_CONST_TKFRZ
   use shr_log_mod                        , only : errMsg => shr_log_errMsg
-  use clm_varpar                         , only : nlevsoi, nlevgrnd, nlevdecomp, ndecomp_cascade_transitions, ndecomp_pools
+  use clm_varpar                         , only : nlevdecomp
```

Comment:
Removed items that are not in use.

Reply:
In the BGC version of this module these are not used. I may need to bring them back if I discover that the MIMICS version of the module requires them. For now though we don't need to worry about this.